createteachers.rb

```diff
@@ -0,0 +1,18 @@
+require './teacher'
+require './person'
+require './nameable'
+
+def create_a_teacher
+  print 'Age: '
+  age = gets.chomp.to_i
+
+  print 'Name: '
+  name = gets.chomp
+
+  print 'Specialization: '
+  specialization = gets.chomp
+
+  @persons.push(Teacher.new(specialization, age, name))
```

Comment:
![image](https://user-images.githubusercontent.com/46839548/213864997-568e020f-468a-4aa0-ae5b-af167065295f.png)

- [ ] Kindly take a look at the error above ☝️ and notice that Age has the wrong value of "Math" because you have mixed up the order of the constructor variables. Please re-arrange it this way to fix the problem 👇 
```ruby
  @persons.push(Teacher.new(age, specialization, name))
```

Reply:
thank you